sonic_platform_base/chassis_base.py

```diff
@@ -197,6 +232,19 @@ def get_module(self, index):
 
         return module
 
+    def get_module_index(self, module_name):
+        """
+        Retrieves module index from the module name
+
+        Args:
+            module_name: A string, prefixed by SUPERVISOR, LINE-CARD or FABRIC-CARD
+            Ex. SUPERVISOR0, LINE-CARD1, FABRIC-CARD5
+
+        Returns:
+            An index of the ModuleBase object in the module_list
```

Comment:
Suggest rewording to specify return value type:
```
An integer, the index of the ModuleBase object in the module_list
```

Reply:
Fixed. Please review.